modules/mvt/docs/api-reference/mapbox-vector-tile-loader.md

```diff
@@ -16,12 +16,32 @@ Loader for the [Mapbox Vector Tile](https://docs.mapbox.com/vector-tiles/specifi
 import {MVTLoader} from '@loaders.gl/mvt';
 import {load} from '@loaders.gl/core';
 
-const data = await load(url, MVTLoader);
+// Point objects with geometry decoded from (0, 0)
+const geometryData = await load(url, MVTLoader);
+
+// GeoJSON Features
+const loaderOptions = {
+  mvt: {
+    geojson: true,
+    tileIndex: {
+      x: 0,
+      y: 0,
+      z: 0
+    }
+  }
+};
+
+const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
 ```
 
 ## Options
 
-N/A
+| Option          | Type                                             | Default      | Description                                                                                                                                                                                                                                                                                     |
+| --------------- | ------------------------------------------------ | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `mvt.geojson`   | Boolean                                          | true         | If `true`, parser will return a flat array of GeoJSON representations of the features with coordinates decoded from provided tile index. If `false`, parser will return a flat array of [Point](https://github.com/mapbox/point-geometry) objects with feature coordinates decoded from (0, 0). |
+| `mvt.tileIndex` | Object ({x: `number`, y: `number`, z: `number`}) | _No default_ | Mandatory if using `geojson` output (It will return null coordinates if omitted). An object containing tile index values (x, y, z) to decode coordinates.                                                                                                                                       |
```

Comment:
Nit: Not sure that returning `null` is the best choice. Ideas: 
- Set `tileIndex` to some default, e.g. (`0, 0, 0`)
- throw an error (or at least emit a warning) if `geojson` is set but `tileIndex` is omitted.

(We could also combine `geojson` and `tileIndex` but I am thinking that all the geospatial loaders should support a `geojson` option so rather keep that as a simple boolean for consistency)